### ISSUE TITLE
Error message is not helpful when the context defined in build doesn't exist

### ISSUE THREAD

**Issue by rberrelleza** [okteto/okteto#2626]
**To Reproduce**
I defined an okteto.yml like this:

```
name: app

build:
  api:
    context: api
  frontend:
    context: frontend
deploy:
  - helm upgrade --install movies chart --set api.image=${OKTETO_BUILD_API_IMAGE} --set frontend.image=${OKTETO_BUILD_FRONTEND_IMAGE}
dev:
  api:
    command: ["bash"]
    forward:
      - 8080:8080
      - 9229:9229
    sync:
      - api:/usr/src/app
  frontend:
    command: npm start
    sync:
      - frontend:/usr/src/app
```

My repo doesn't have an `api` or `frontend` folder. 

When deploying, I get the following error:

```
Found okteto manifest on /okteto/src/okteto.ymlUnmarshalling manifest...Unmarshalling compose...x Invalid manifest:
- line 18: field dependencies not found in type model.deployInfoRaw
- line 20: field dev not found in type model.deployInfoRaw
See https://okteto.com/docs/reference/manifest/ for details
exit status 1
```

**Expected behavior**
I'd expect a clear error like: "folder API doesn't exist" or something like that.

**Comment by teresaromero** [comment]
i've tested this scenario but not sure if its reproducible anymore. at `build` if i use for `context` a value for a folder that does not exist, i get the following error

```
❯ okteto deploy --build
 i  Using teresa @ okteto.teresa.dev.okteto.net as context
 i  Building image for service 'api'
 x  Invalid build context: 'pepe' does not exist
```

@rberrelleza which version does this happened? can we check this is still not working?

**Comment by teresaromero** [comment]
@rberrelleza can't reproduce at 2.4.1.
tested with latest movies repo, changing the context of frontend to a non existent folder

shall we close as resolved? 

@pchico83  i've seen that when multiple images are built the contexts are not checked before starting, so the process could build n images and fail when it gets to the invalid context. shall we optimise this by checking the contexts prior to the build execution? just an idea 

cc: @jLopezbarb @jmacelroy

**Comment by pchico83** [comment]
> @rberrelleza can't reproduce at 2.4.1. tested with latest movies repo, changing the context of frontend to a non existent folder
> 
> shall we close as resolved?
> 
> @pchico83 i've seen that when multiple images are built the contexts are not checked before starting, so the process could build n images and fail when it gets to the invalid context. shall we optimise this by checking the contexts prior to the build execution? just an idea
> 
> cc: @jLopezbarb @jmacelroy

@teresaromero yes, all the validation should happen before starting any build or deploy command

**Comment by jmacelroy** [comment]
@teresaromero if you can't reproduce on later releases then let's close. Can you please open a new issue for the validation optimization and put the info you have on it there?

**Comment by teresaromero** [comment]
Added this ^^ @jmacelroy

**Comment by jLopezbarb** [comment]
@teresaromero I can't reproduce it either. Let's close it and reopen if we can reproduce it
